src/codegen/riscv64/constants-riscv64.h

```diff
@@ -833,6 +846,11 @@ class InstructionGetters : public T {
     return this->Bits(kRvcRs2Shift + kRvcRs2Bits - 1, kRvcRs2Shift);
   }
 
+  inline int RvcRdsValue() const {
+    DCHECK(this->IsShortInstruction());
+    return 0b1000 + this->Bits(kRvcRdsShift + kRvcRdsBits - 1, kRvcRdsShift);
+  }
+
```

Comment:
This method should be removed. Because the position of rd' register may be not same (see CIW and CL type). Using this method to get rd' would cause a conflict. Please use `RvcRs1sValue()` or `RvcRs2sValue()` to get different rd' according to instruction's type.

![图片](https://user-images.githubusercontent.com/17661043/93956898-1ce25600-fd8e-11ea-9645-b8d925a38e41.png)